lessonbuilder/tool/src/webapp/css/Simplepagetool.css

```diff
@@ -198,7 +198,7 @@ div.action span .nextbutton, .Mrphs-siteHierarchy span .nextbutton{
 
 }
 .act .deleteButton {
-    float: none
+    float: right
```

Comment:
I'm still not sure the far right is the correct place for this on wider screens, but I suppose that is more of a responsive design issue. I think it is certainly more visible as a button though.